src/main/java/com/minecolonies/blockout/controls/TextField.java

```diff
@@ -24,6 +24,7 @@
      * Texture resource location.
      */
     private static final ResourceLocation TEXTURE           = new ResourceLocation("textures/gui/widgets.png");
+    private static final int RECT_COLOR = -3_092_272;
     //  Attributes
     protected            int              maxTextLength     = 32;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 32 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS109)